pkg/containertools/command.go

```diff
@@ -41,9 +41,13 @@ func (r *ContainerCommandRunner) GetToolName() string {
 
 // Pull takes a container image path hosted on a container registry and runs the
 // pull command to download it onto the local environment
-func (r *ContainerCommandRunner) Pull(image string) error {
+func (r *ContainerCommandRunner) Pull(image string, skipTLS bool) error {
 	args := []string{"pull", image}
 
+	if r.containerTool.String() == "podman" {
```

Comment:
Shouldn't this check that `skipTLS` is true before adding the arg? Also, are the tests not running against Docker?

Reply:
Well, that's a good catch.

Reply:
They do run against docker, that particular parameter only exists (and is only needed) by podman.

Reply:
Oh, ok. Didn't see the config you're generating in the test script.